Makefile

```
@@ -0,0 +1,15 @@
+
+build:
+	docker compose build
+	docker compose run --rm go go mod tidy
+run:
+	docker compose up db -d
+	docker compose run --rm go go run main.go
+run-go:
+	docker compose up db -d
+	docker compose run --rm go go run main.go
```

```suggestion
run:
	docker compose up db -d
	docker compose up go 
```
`run`コマンドと`run-go`同じなので、一つ消してよさそう。
`docker compose run --rm go go run main.go`だとAirが走らないので、docker compose up goでお願いします。